x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats_percentile.csv-spec

```diff
@@ -143,6 +143,21 @@ MEDIAN(salary):double | MEDIAN_ABSOLUTE_DEVIATION(salary):double
 // end::median-absolute-deviation-result[]
 ;
 
+medianAbsoluteDeviationFold
```

Comment:
Please, either change the name of the file or create a new file with all the m_a_d tests and leave `stats_percentile.csv-spec` with only percentile related tests.

Reply:
Or move every mv_m_a_d and m_a_d in the same file.

Reply:
Created a new file for MAD (Better a file per function I think, but still a lot to migrate)